.config/nextest.toml

```diff
@@ -9,3 +9,14 @@ fail-fast = false
 # Write to output for persistence to CircleCI
 [profile.ci.junit]
 path = "junit.xml"
+
+# Integration tests require more than one thread. The default setting of 1 will cause too many integration tests to run
+# at the same time and causes tests to fail.
+[[profile.default.overrides]]
+filter = 'package(apollo-router) & kind(test)'
+threads-required = 2
+
+# Scaffold test takes all the test threads as it runs rustc.
+[[profile.default.overrides]]
+filter = 'package(apollo-router-scaffold)'
```

Comment:
Does this need to be called out as an exception that exists somewhere around the actual `apollo-router-scaffold` file?

Reply:
[169faa2](https://github.com/apollographql/router/pull/5109/commits/169faa27ee4b3b1b904a7922f3b1c254a3e0f68b)